server/events/vcs/gitlab_client.go

```diff
@@ -577,8 +577,17 @@ func (g *GitlabClient) MarkdownPullLink(pull models.PullRequest) (string, error)
 	return fmt.Sprintf("!%d", pull.Num), nil
 }
 
-func (g *GitlabClient) DiscardReviews(_ models.Repo, _ models.PullRequest) error {
-	// TODO implement
+// DiscardReviews discards all reviews on a pull request
+// This is only available with a bot token and otherwise will return 401 unauthorized
+func (g *GitlabClient) DiscardReviews(repo models.Repo, pull models.PullRequest) error {
```

Comment:
Please add debug logs following the pattern of the other GitLab client functions that call the GitLab API.